src/main/resources/static/api.yaml

```diff
@@ -2172,3 +2242,33 @@ components:
       properties:
         refreshToken:
           type: string
+    OtpVerificationRequest:
+      type: object
+      properties:
+        username:
+          type: string
+    OtpVerificationResponse:
+      type: object
+      properties:
+        sessionId:
+          type: string
+    OtpPermitRequest:
+      type: object
+      properties:
+        sessionId:
+          type: string
+        otp:
+          type: string
+    SessionResponse:
+      type: object
+      properties:
+        access_token:
+          type: string
+        expires_in:
+          type: int
+        refresh_expires_in:
+          type: int
+        refresh_token:
+          type: string
+        token_type:
+          type: string
```

Comment:
Don't we have this already? casing also wrong.

Reply:
I don't think we have it already, that's why added. which casing? camelcase or lowercase. If camelcase, it's the way current response is.

Reply:
```
   AuthResponse:
      type: object
      properties:
        accessToken:
          type: string
          example: eyJhbGciOiJSUzI1Ni.IsInR5cCIgOiAiSldUIiwia2lkIiA6ICJrVVp.2MXJQMjRyYXN1UW9wU2lWbkdZQUZIVFowYVZGVWpYNXFLMnNibTk0In0
        expiresIn:
          type: integer
          example: 1800
          description: In Minutes
        refreshExpiresIn:
          type: integer
          example: 1800
          description: In Minutes
        refreshToken:
          type: string
          example: eyJhbGciOiJSUzI1Ni.IsInR5cCIgOiAiSldUIiwia2lkIiA6ICJrVVp.2MXJQMjRyYXN1UW9wU2lWbkdZQUZIVFowYVZGVWpYNXFLMnNibTk0In0
        tokenType:
          type: string
          example: bearer
```

What's different in your type?

Reply:
camelcasing is not representative of the actual response right..? FTR I didn't see AuthResponse..

Reply:
Removed redundant addition

Reply:
our response should always be in camelcase.

Reply:
Yes. I've updated the api to use the existing AuthResponse only.